android/res/xml/prefs_main.xml

```diff
@@ -178,14 +178,24 @@
     </PreferenceScreen>
   </androidx.preference.PreferenceCategory>
   <androidx.preference.PreferenceCategory
```

Comment:
Format please your  changes, when you apply code style scheme.

Reply:
Removed preference altogether, no longer required.